pom.xml

```diff
@@ -109,6 +124,10 @@
             <failOnWarnings>true</failOnWarnings>
           </configuration>
         </plugin>
+        <plugin>
```

Comment:
You still have changes to nexus-staging-maven-plugin's config.  I'm not expecting to see any